### ISSUE TITLE
Ads no longer getting served to app

### ISSUE THREAD

**Issue by rolinger** [floatinghotpot/cordova-admob-pro#590]
Over the last few weeks I have noticed my app is no longer getting any ads displayed..on Android and iOS both.  I have been running this plugin for about 1.5 years without any issue and now its not working.  I have seen plenty of other response to similar posts "looks like you there is no ad to serve" - just having a hard time believing that is the real reason here.   Esp after 1.5 years of serving hundreds of thousands of ads...suddenly my app is no longer worthy of getting matched ads.

This is the error I am getting:
2017-08-28 21:19:48.866658-0400 [1868:1562871] AdMob, onAdFailLoad, {'adNetwork':'AdMob','adType':'banner','adEvent':'onAdFailLoad','error':1,'reason':'Request Error: No ad to show.'}
2017-08-28 21:20:18.982214-0400 [1868:1573294] <Google:HTML> You are currently using **version 7.21.0 of the SDK.** Please consider updating your SDK to the most recent SDK version to get the latest features and bug fixes. The latest SDK can be downloaded from http://goo.gl/iGzfsP. A full list of release notes is available at https://developers.google.com/admob/ios/rel-notes.

The version of the plugin I am running is: 2.29.21.  the above error message is showing the SDK as out of date...could this be related?

**Comment by floatinghotpot** [comment]
Google has released SDK iOS v7.22.0 on 2017‑08‑08. 

The plugin needs to be upgraded. 

Please update to: `cordova-plugin-admobpro@2.29.22`

**Comment by rolinger** [comment]
Ok....will give that a shot...I had actually upgraded the plugin using @latest and that installed 2.29.22.  Hopefully thats it.

**Comment by rolinger** [comment]
After upgrading the plugin on both Android and iOS platforms, the Android still gives me the same error but I am getting a new error now on iOS:

> <Google> To get test ads on this device, call: request.testDevices = @[ @"048c4e4f84e224620324d5fabdae779a" ];

> AdMob, onAdFailLoad, {'adNetwork':'AdMob','adType':'banner','adEvent':'onAdFailLoad','error':1,'reason':'Request Error: No ad to show.'}

It looks like something has drastically changed.  What does this new message mean?

**Comment by rolinger** [comment]
When I logged into my AdMob account there was an alert screen across the top that said "Ad units are suspended until you have verified your pin" (or something similar to that).  So I tracked down the postcard google had mailed me and entered the PIN - I assume it took the pin as it just greyed out all the buttons - but it never said "confirmed"...or anything else.   I closed that window and returned to the main screen and about an hour later ads started showing up again in my app.
